openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -21,30 +21,29 @@ <h2 class="step__title">
                         {% endif %}
                     </span>
                 </button>
-            </h2>
+            </h4>
         </header>
 
         <div class="ui-slidable__content" id="oa_grade_{{ xblock_id }}_content" aria-labelledby="oa_grade_{{ xblock_id }}">
             <div class="wrapper--step__content">
                 <div class="step__content">
                     <article class="submission__answer__display step__content__section">
-                        <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
```

Comment:
I think this is just the same thing as I commented on below-- the variable "translated_label" is being defined and passed into a template, where it is indeed used in an h5. Got it.